app/_data/docs_nav_ee_0.31-x.yml

```diff
@@ -28,9 +28,9 @@
   items:
     - text: Canary Release Plugin
       url: /plugins/canary-release
-    - text: Rate Limiting Plugin
+    - text: Rate Limiting Advanced Plugin
```

Comment:
The sidebar looks a bit funky saying "Plugin after every item", as it looks like we're not consistent. Can we be consistent and remove "plugin" for each item?

Reply:
I didn't add "Plugin" - removing it sounds fine to me, but perhaps not as part of this PR?

Reply:
When you add "advanced" to this plugin, it becomes the first plugin to jump to a new line. I'm saying we could get around that by removing "plugin" from each item since some of them already omit.